Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,372 @@
+```console
+ceph dashboard set-grafana-api-url <grafana_url>
+ceph dashboard set-grafana-api-ssl-verify <False|True>
+ceph dashboard set-alertmanager-api-host <alert manager url>
+ceph dashboard set-alertmanager-api-ssl-verify <False|True>
+ceph dashboard set-prometheus-api-host <prometheus url>
+ceph dashboard set-prometheus-api-ssl-verify <False|True>
+```
+
+## Preparation
+
+Clone the Rook repository to have access to the different resource files used in the next steps:
+```console
+git clone --single-branch --branch {{ branchName }} https://github.com/rook/rook.git
+cd rook/cluster/examples/kubernetes/ceph/monitoring
+```
+
+## Ceph Cluster
+
+Dashboard, prometheus manager modules and Prometheus monitoring must be enabled in your cluster CRD:
+```yaml
+mgr:
+...
+  modules:
+    ...
+  - name: prometheus
+    enabled: true
+...
+dashboard:
+  enabled: true
+  ssl: true
+...
+monitoring:
+  enabled: true
+  rulesNamespace: rook-ceph  
+```
+
+## Prometheus Operator
+
+The Prometheus operator allows you to set up a Prometheus instance in you kubernetes cluster. Follow the instructions provided in [Prometheus Monitoring](https://github.com/rook/rook/blob/{{ branchName }}/Documentation/ceph-monitoring.md).
+
+Below a brief summary of the things you need to do:
+
+[Install the Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator#quickstart)
+
+Go to the folder where the different resource files for the monitoring stack are stored
+```console
+cd rook/cluster/examples/kubernetes/ceph/monitoring
+```
+Create and configure the Prometheus instance 
+```console
+kubectl create -f service-monitor.yaml
+kubectl create -f prometheus.yaml
+kubectl create -f prometheus-service.yaml
+```
+
+With these steps you must have a running instance of Prometheus configured to scrape data from the Ceph cluster through the Ceph Prometheus manager module
+
+To verify that prometheus is running you can try to access the prometheus instance.
+
+Execute:
+```console
+echo "http://$(kubectl -n rook-ceph -o jsonpath={.status.hostIP} get pod prometheus-rook-prometheus-0):30900"
+```
+>```
+>http://192.168.122.70:30900
+>```
+If you use the output URL in your browser, you should see the Prometheus monitoring website.
+
+![Prometheus Monitoring Website](media/prometheus-monitor.png)
+
+## Prometheus Ceph manager module
+
+The source of the Ceph cluster metrics is this manager module, by default is enabled, and the Prometheus operator service monitor configures Prometheus to scrape data from it.
+
+In order to avoid monitoring issues just verify that the scrape intervals are the same in Prometheus and in the Ceph manager module. Both intervals must be the same, depending on your monitoring needs, a value between 5 and 30 is correct.
```

Comment:
```suggestion
In order to avoid monitoring issues just verify that the scrape intervals are the same in Prometheus and in the Ceph manager module. Both intervals must be the same. Depending on your monitoring needs, a value between 5 and 30 is correct.
```